packages/passport-interface/src/PassportInterface.ts

```diff
@@ -125,11 +125,40 @@ export function constructZupassPcdProveAndAddRequestUrl<
   return `${zupassClientUrl}#/add?request=${eqReq}`;
 }
 
-export const getAnonTopicNullifier = (
-  chatId: number,
-  topicId: number
-): bigint => {
-  return BigInt(
-    "0x" + sha256(JSON.stringify({ chatId, topicId })).substring(0, 16)
-  );
+export const getAnonTopicNullifier = (): bigint => {
+  return BigInt("0x" + sha256("anon_message").substring(0, 16));
 };
+
+export enum PayloadType {
+  RedirectTopicData = "topic-data",
+  NullifierHash = "nullifier-hash",
+  AnonTopicDataPayload = "anon-topic-data-payload"
```

Comment:
we don't need it to be in this union type, maybe make it into a separate issue since it's not blocking

Reply:
#1154